docs/index.html

```diff
@@ -10,6 +10,7 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width">
     <link rel="stylesheet" href="blueprint-landing.css">
+    <link rel="icon" href="resources/img/favicon.png" sizes="32x32" type="image/png">
```

Comment:
Should we move `landing/src/resources` to `landing/resources` in this PR, or in a separate one?